plugins/CoreHome/angularjs/report-export/reportexport.directive.js

```diff
@@ -185,7 +185,7 @@
                         }
                     }
 
-                    exportUrlParams.token_auth = piwik.token_auth;
+                    exportUrlParams.token_auth = _pk_translate('CoreHome_EnterYourToken');
```

Comment:
btw @flamisz when clicking on the export button then I think we'd still want the correct token auth to be used. 
![image](https://user-images.githubusercontent.com/273120/108152192-60729d80-713d-11eb-9a8b-90caf109327d.png)

Only in the textarea we would want to show the `ENTER_YOUR_TOKEN_AUTH_HERE`.

When clicking on the export button currently it says 
![image](https://user-images.githubusercontent.com/273120/108152262-86983d80-713d-11eb-9d7f-20c6c3a3362a.png)

vs it should show the exported data.

Also additionally, the parameter `&force_api_session=1` should not be shown in the text area 
![image](https://user-images.githubusercontent.com/273120/108152321-acbddd80-713d-11eb-8160-aadde11a94a1.png)

Sorry this wasn't mentioned in the issue probably. Otherwise it would compare the token against a "session" token but not against a proper token from the configured auth tokens in the security page AFAIK. So this parameter would need to be removed in the textarea but kept when clicking on the "export" button.

Reply:
Yeah sure, you're right, the button needs the token. I fix that and delete the `&force_api_session=1` parameter from the url in the textarea (only there).